source/sass/main.scss

```diff
@@ -148,15 +291,211 @@ header {
+
+  h4 {
+    color: #999;
+    margin-bottom: 1.3em;
+    font-size: 1.5em;
+    font-weight: 300;
+  }
+
+  li {
+    font-size: 1em;
+    line-height: 1.2em;
+
+    a {
+      color: #30aac6;
+    }
+
+  }
+}
+
+footer {
+  margin-top: 100px;
+}
+
+.mofooter {
+  background: #4d4f53;
+  color: $white;
+  text-align: center;
+  padding: 2.8125rem 0.625rem;
+
+  .icon-container {
+    margin: 1.25rem 0;
+
+    .icon {
+      cursor: pointer;
+      display: inline-block;
+      font-weight: 300;
+      font-size: 1rem;
+
+      img {
+        width: 1.25rem;
+      }
+
+      .footer-icon {
+        opacity: 0.5;
+      }
+
+      a {
+        color: $white;
+        opacity: 0.5;
+        text-decoration: none;
+      }
+
+      &:hover {
+        a,
+        .footer-icon {
+          transition: opacity .15s ease-out;
+          opacity: 1;
+        }
+      }
+    }
+  }
+
+  .footer-content {
+    max-width: 35.625rem;
+    margin: 0 auto;
+
+    p {
+      font-size: 0.875rem;
+      color: rgba($white, 0.5);
+      line-height: 1.5em;
+      margin: 0;
+
+      a {
+        color: rgba($white, 0.8);
+        line-height: inherit;
+
+        &:hover {
+          color: rgba($white, 1);
+        }
+      }
+    }
+  }
+
+  .icons {
+    margin-bottom: 1.25rem;
+  }
+
+  .logo {
+    margin-bottom: 1.25rem;
+    padding-bottom: 1.25rem;
+    border-bottom: 0.0625rem solid #666;
+    display: inline-block;
+
+    @media screen and (min-width: $bp-md) {
+      border-bottom: 0;
+      border-right: 0.0625rem solid #666;
+      padding-bottom: 0;
+      padding-right: 1.25rem;
+      margin-bottom: 0;
+      margin-right: 1.25rem;
+    }
+  }
+
+  @media screen and (max-width: $bp-md) {
+    .icon-container {
+      &:first-child {
+        margin-top: 0;
+      }
+
+      &:last-child {
+        margin-bottom: 2.8125rem;
+      }
+
+      .footer-icon {
+        display: block;
+        margin: 0.375rem auto 0.375rem;
+      }
+    }
+  }
+
+  @media screen and (min-width: $bp-md) {
+    .footer-content {
+      display: flex;
+      align-items: center;
+      flex-shrink: 0;
+      justify-content: center;
+
+      .footer-icon {
+        margin-left: 0.625rem;
+      }
+
+      p {
+        text-align: left;
+        width: calc(100% - 10.375rem);
+      }
+    }
+    .icon-container {
+      display: inline-block;
+      margin: 0 1.125rem;
+
+      .icon {
+        display: flex;
+        align-items: center;
+        flex-shrink: 0;
+
+        .footer-icon {
+          margin: 0 0.625rem 0 0;
+          display: inline-block;
+        }
+      }
+    }
+  }
+}
+
+.partner-logos {
+  margin-bottom: 20px;
 
   a {
-    text-decoration: underline;
+    max-width: 100%;
+    display: block;
+    margin: 0 auto;
+    transition: opacity .1s ease-out;
+    text-align: center;
+    height: 100px;
+    line-height: 100px;
+
+    img {
+      max-height: 80px;
+      max-width: 85%;
+      vertical-align: middle;
+
+      &[src*="itu"] {
```

Comment:
Styling based on attribute values is not very advisable unless you absolutely have to. It's easily breakable and harder to debug.

Can you use class names instead?